components/common/Error500.tsx

```diff
@@ -0,0 +1,21 @@
+import MainLayout from '../layout/MainLayout';
+import ResultsRetryButton from '../results/ResultsRetryButton';
+
+import CenteredCard from './CenteredCard';
+import CenteredCardContent from './CenteredCardContent';
+
+export default function Error500(): JSX.Element {
```

Comment:
To not use the 500 magic number, what could be a more descriptive component name? ServerError might be slightly better. Do we know anything more specific about what kinds of errors this shows?

Reply:
I just added a catch all error that should happen in case of react error, for now it never triggered so let's see!

Reply:
I'll rename to ErrorRuntime